### DISCUSSION TITLE
Introduce reverse proxy functionality in Dirigible

### DISCUSSION THREAD

**Discussion by iliyan-velichkov** [eclipse/dirigible#2999] [Ideas]
Consider to introduce reverse proxy functionality in Dirigible using [Netflix Zuul](https://cloud.spring.io/spring-cloud-netflix/multi/multi__router_and_filter_zuul.html).

**Comment by delchev** [comment]
Looks promising! Maybe we can introduce `*.proxy` definition files for configurations instead of `application.properties` and a synchronizer which can dynamically expose the proxy endpoints
